src/blocks/index.tsx

```diff
@@ -47,7 +46,7 @@ const blockComponents = {
   header_block: HeaderBlock,
   header_block_brand_pivot: HeaderBlockBrandPivot,
   about_us_hero_block: AboutUsHeroBlock,
-  accordion_block: AccordionBlock,
+  accordion_block: AccordionBlockBrandPivot,
```

Comment:
Still have some page where this is used so will just map this to the new component until it's removed in the CMS